### ISSUE TITLE
"State" parameters

### ISSUE THREAD

**Issue by rtimms** [pybamm-team/BPX#4]
- Should we store information on "battery state" in a separate field? E.g. initial concentrations, initial temperature 
- Add min/max electrode stoichiometries (and validate using OCV and min/max voltage)
- Store time, current, voltage, heat transfer coefficient (if applicable) in "experiment" field separate from cell parameters

**Comment by ejfdickinson** [comment]
"Battery state" could also contain "SOC" for use in place of initial concentrations. We should specify that supplying this information is optional.

**Comment by ejfdickinson** [comment]
@rtimms Yes, if that is not an unreasonable burdensome proposal from a technical standpoint, we could do that.
